### DISCUSSION TITLE
Zen 55 (smoke detector module) alerts whenever the ZWave JS integration restarted.

### DISCUSSION THREAD

**Discussion by Doni49** [zwave-js/zwave-js#7299] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
I have a Zooz Zen 55 smoke detector.   Whenever the ZWave JS integration is restarted (whether the server is rebooted or the integration is just reloaded), the detector status changed to "detected".  I've reached out to Zooz and they tell me they can not reproduce it on their end and that it has to be something in the integration.

What did you expect to happen instead?
When the integration is reloaded, I should not be getting an alert from my smoke detector.

 I am using Node Red for automations.   I have a trigger node set to fire when binary_sensot.fire_detector_smoke_detected is true.  The only thing it currently does is send a notification to my cell phone.

Anything else we should know?

### Software versions

Driver (node-zwave-js): ... N/A

Z-Wave JS UI: ... N/A

Home Assistant Z-Wave Integration: ... N/A

Home Assistant Z-Wave JS Addon: ... 0.8.0

ioBroker.zwave2 Adapter: ... N/A

If you are using something non-standard, tell us here: ...

### Z-Wave Controller (Stick/Dongle/...)

70/800 Series dongle by Silicon Labs
Firmware:  7.18.3

### Device information

ZOOZ
Zen 55LR
Node 49

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[logs.txt](https://github.com/user-attachments/files/17464617/logs.txt)

Also, here's the reply email from Zooz.

Thank you for your patience while our technicians reviewed the details.

We are not able to replicate your issue in our test set up; the ZEN55 is not capable of sending any notifications aside from those triggered by the physical hardwired sensor. It's possible that with the plugin restart, HA is asking the ZEN55 for something.

 It looks like it's an issue within the Home Assistant platform.

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment, marked as answer]
This is a question for the HA community forums or Discord, since this is strictly related to how you automate.
AFAIK the reason is that the entities are marked unavailable in HA while Z-Wave JS is not running and then the status is restored. So you have to ignore the unavailable status somehow.
